tests/rules/space-before-brace.js

```diff
@@ -2,9 +2,12 @@
 
 var lint = require('./_lint');
 
-var file = lint.file('space-before-brace.scss');
+//////////////////////////////
```

Comment:
I feel this rule should probably provide a warning itself if enabled on a Sass syntax file? What do you reckon?